example/viam_robot_example_app/lib/main.dart

```diff
@@ -223,7 +172,7 @@ class _MyHomePageState extends State<MyHomePage> {
                           Text(_robot.resourceNames.where((element) => element.type == resourceTypeComponent).join('\n')),
                         ])
                       : _loading
-                          ? PlatformCircularProgressIndicator()
+                          ? const CircularProgressIndicator()
```

Comment:
Perhaps use `CircularProgressIndicator.adaptive()` to keep it more native (since you're removing all the platform native stuff)

Reply:
Ok i'll add that. Just trying to simplify the examples as much as possible. `.adaptive()` on any standard flutter widgets doesn't feel to intrusive.